app/Http/Controllers/UserController.php

```diff
@@ -40,4 +43,42 @@ public function handleLike(Request $request)
 
         return json_encode(['success' => trans('pages.success')]);
     }
+
+    public function comment(CommentRequest $request, $slug)
+    {
+        try {
+            $news = News::where('slug', $slug)->firstOrFail();
+        } catch (ModelNotFoundException $exception) {
+            return redirect()->back()->withErrors(trans('pages.error'));
+        }
+        if ($request->parent_id != null) {
+            Comment::create([
+                'content' => $request->comment_content,
+                'user_id' => Auth::id(),
+                'news_id' => $news->id,
+                'parent_id' => $request->parent_id,
+            ]);
+        } else {
+            Comment::create([
+                'content' => $request->comment_content,
+                'user_id' => Auth::id(),
+                'news_id' => $news->id,
+            ]);
+        }
```

Comment:
```suggestion
            Comment::create([
                'content' => $request->comment_content,
                'user_id' => Auth::id(),
                'news_id' => $news->id,
                'parent_id' => $request->parent_id,
            ]);
```
Đoạn này ko cần check, cứ create cả thôi